{"uuid": "29d5bc69-dab7-4212-a297-3f84fce668f4", "counter": 53, "stdout": "\u001b[0;31mfatal: [demo03]: FAILED! => {\"changed\": false, \"msg\": \"One or more supplied key could not be found in the database.\"}\u001b[0m\r\n\u001b[0;36m...ignoring\u001b[0m", "start_line": 57, "end_line": 59, "runner_ident": "630e1c67-15b7-4206-bf0b-2acc198a6083", "event": "runner_on_failed", "pid": 256548, "created": "2025-05-16T19:57:28.056759+00:00", "parent_uuid": "005056b7-f67d-196a-5680-000000000014", "event_data": {"playbook": "/tmp/tmp3w5j5ydz.yml", "playbook_uuid": "05688121-c63f-481c-8e7a-1879289a8e10", "play": "Instalaci\u00f3n y configuraci\u00f3n de Pure-FTPd para hosts", "play_uuid": "005056b7-f67d-196a-5680-000000000002", "play_pattern": "demo03", "task": "Verificar si el usuario FTP existe", "task_uuid": "005056b7-f67d-196a-5680-000000000014", "task_action": "getent", "resolved_action": "ansible.builtin.getent", "task_args": "", "task_path": "/tmp/tmp3w5j5ydz.yml:149", "host": "demo03", "remote_addr": "demo03", "res": {"msg": "One or more supplied key could not be found in the database.", "invocation": {"module_args": {"database": "passwd", "key": "ftpuser", "fail_key": true, "service": null, "split": null}}, "_ansible_no_log": false, "changed": false}, "start": "2025-05-16T19:57:27.078254+00:00", "end": "2025-05-16T19:57:28.056375+00:00", "duration": 0.978121, "ignore_errors": true, "event_loop": null, "uuid": "29d5bc69-dab7-4212-a297-3f84fce668f4"}}